CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Bug fixes
+
+* [#237](https://github.com/rubocop/rubocop-ast/pull/237) Fix `#macro?` for numblock nodes ([@gsamokovarov][])
```

Comment:
Can you use the new changelog entry method instead of editing CHANGELOG.md directly?
https://github.com/rubocop/rubocop/blob/v1.33.0/CONTRIBUTING.md#creating-changelog-entries

Reply:
Surely, Kichi, thanks for noticing. I have run `rake changelog:new` and added the changelog entry for this change.